lib/TPP/GPU/GpuPipeline.cpp

```diff
@@ -70,6 +70,31 @@ GpuType parseGpuOption(StringRef gpuStr) {
   return *type;
 }
 
+struct GpuOptions {
+  std::string triple;
+  std::string chip;
+  std::string features;
+};
+
+GpuOptions getGpuOptions(GpuType gpuType) {
+  GpuOptions options;
+
+  switch (gpuType) {
+  case GpuType::Cuda: {
+    options.triple = "nvptx64-nvidia-cuda";
+    options.chip = "sm_70";
+    options.features = "+ptx60";
+    break;
+  }
+  case GpuType::Vulkan: {
```

Comment:
We'll change this to `SYCL` or some other non-Vulkan thing, right?

Reply:
Yes, I intend to completely replace Vulkan with SYCL.